core/src/themes/ios/default.tokens.ts

```diff
@@ -12,10 +14,16 @@ export const defaultTheme: DefaultTheme = {
   palette: {
     light: lightTheme,
     dark: darkTheme,
+    highContrast: highContrastTheme,
+    highContrastDark: highContrastDarkTheme,
   },
 
   fontFamily: '-apple-system, BlinkMacSystemFont, "Helvetica Neue", "Roboto", sans-serif',
 
+  color: {
+    'overlay-bg': 'var(--ion-overlay-background-color, var(--ion-color-gray-100))', // Available only in iOS
```

Comment:
`--ion-overlay-background-color` and this variable are very similar in naming. Is it possible to get rid of `--ion-overlay-background-color` since it's redundant? I left it in for backwards compatibility.